packages/base/src/hooks/useDeprecateRenderMethods.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useMemo } from 'react';
+import { deprecationNotice } from '../utils';
+
+export const useDeprecateRenderMethods = (renderMethod, renderMethodName, slot, slotName) => {
```

Comment:
I think this one can be simplified
```suggestion
export const useDeprecateRenderMethods = (props, renderMethodName, slotName) => {
```